float64[] target_angle_rad

int16 SIDE_SINGLE = 0
int16 SIDE_PORT = 0
int16 SIDE_STBD = 1